lib/processing.py

```diff
@@ -10,7 +10,7 @@
     try:
         config = yaml.load(file_stream)
     except yaml.YAMLError as exception:
-        print(exception)
+        raise exception
```

Comment:
Oh, I think you can just run `config = yaml.load(file_stream)`, and then if there's an error it'll kill the process; no need for `try` and `except` and `raise`.